maps/away/ships/dominia/dominian_science_vessel/dominian_science_vessel.dm

```diff
@@ -0,0 +1,145 @@
+/datum/map_template/ruin/away_site/dominian_science_vessel
+	name = "Dominian Science Vessel"
+	description = "Based on the Lammergeier-class corvette, this vessel has been repurposed by House Volvalaad for long range survey and scientific tasks. Due to its repurposement, the vessel features an enlarged hangar and shuttle, as well as scientific labs and a smaller defensive armament."
+	suffixes = list("ships/dominia/dominian_science_vessel/dominian_science_vessel.dmm")
+	sectors = list(ALL_TAU_CETI_SECTORS, ALL_COALITION_SECTORS, SECTOR_BADLANDS)
+	spawn_weight = 1
+	ship_cost = 1
+	id = "dominian_science_vessel"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/dominian_science_shuttle)
+
+	unit_test_groups = list(3)
+
+/singleton/submap_archetype/dominian_science_vessel
+	map = "Dominian Science Vessel"
+	descriptor = "Based on the Lammergeier-class corvette, this vessel has been repurposed by House Volvalaad for long range survey and scientific tasks. Due to its repurposement, the vessel features an enlarged hangar and shuttle, as well as scientific labs and a smaller defensive armament."
+
+//ship stuff
+
+/obj/effect/overmap/visitable/ship/dominian_science_vessel
+	name = "Dominian Science Vessel"
+	class = "HVS"
+	desc = "Based on the Lammergeier-class corvette, this vessel has been repurposed by House Volvalaad for long range survey and scientific tasks. Due to its repurposement, the vessel features an enlarged hangar and shuttle, as well as scientific labs and a smaller defensive armament."
+	icon_state = "lammergeier"//td
+	moving_state = "lammergeier_moving"//td
+	colors = list("#df1032", "#d4296b")
+	designer = "Zhurong Naval Arsenal, Empire of Dominia"
+	volume = "36 meters length, 67 meters beam/width, 18 meters vertical height"
+	drive = "Low-Speed Warp Acceleration FTL Drive"
+	weapons = "Single wingtip-mounted extruding medium-caliber ballistic armament, aft obscured flight craft bay"
+	sizeclass = "Explorer-class Science Vessel"
+	shiptype = "Survey and scientific research"
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	initial_restricted_waypoints = list(
+		"Dominian Science Shuttle" = list("nav_hangar_dominian_science_vessel")
+	)
+
+	initial_generic_waypoints = list(
+		"nav_dominian_science_vessel_1",
+		"nav_dominian_science_vessel_2",
+		"nav_dominian_science_vessel_3",
+		"nav_dominian_science_vessel_4",
+		"nav_dominian_science_dock_port",
+		"nav_dominian_science_dock_aft"
+	)
+
+	invisible_until_ghostrole_spawn = TRUE
+
+/obj/effect/overmap/visitable/ship/dominian_science_vessel/New()
+	designation = "[pick("Vanguard", "Voyager", "Explorer", "Owl", "Ocular")]"//td
```

Comment:
not a hard requirement / request, but would be nice to have some more possible names - even better if they were to have dominian theme and vibe